src/sentry/integrations/slack/message_builder/base/base.py

```diff
@@ -38,10 +45,26 @@ def _build(
 
         if title:
             kwargs["title"] = title
+            if title_link:
+                kwargs["title_link"] = title_link
+
+        if actions:
+            kwargs["actions"] = [
+                {
+                    "text": action.label,
+                    "name": action.label,
+                    "url": action.url,
+                    "style": action.style or "default",
+                    "type": "button",
+                }
+                for action in actions
```

Comment:
@mgaeta IMO, if we aren't going to have a class method `as_slack`, we should still make a method that replicates the functionality rather than embedding the mapping logic in `SlackMessageBuilder`